### DISCUSSION TITLE
Missing/incorrect accessibility features on Splitter

### DISCUSSION THREAD

**Discussion by shawnharvell** [primefaces/community#1042] [PrimeReact] [answered]
I refer to the detailed specs outlined here: https://www.w3.org/WAI/ARIA/apg/patterns/windowsplitter/

the incorrect one is the use of aria-orientation which while the component prop refers to the orientation of the panels themselves, the aria-orientation should be the orientation of the separator, i.e., always the inverse of the component orientation.

the missing are:

role="separator" on the separator
aria-valuemin and aria-valuemax on the separator
aria-controls on the separator with a value that points to the primary panel
props should be added to the component to allow setting of aria-label and/or aria-labeledby on the separator

and then ideally add to the keyboard handling support for the Home, End and Enter keys

**Comment by melloware** [comment]
@shawnharvell please open a github ticket: https://github.com/primefaces/primereact/issues

**Reply by shawnharvell** [reply, marked as answer]
@melloware  thank you https://github.com/primefaces/primereact/issues/5849
